docs/getting-started/summaryofchanges.md

```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
 
 #### Zowe App Server
 
-- In this version, the app server can now use more than one CA [#128](https://github.com/zowe/zlux-app-server/pull/128)
-- Removed usage of ngx-color [#235](https://github.com/zowe/zlux-app-manager/pull/235)
-- The `dispatcher.invokeAction` method now returns promise, which provides the ability to wait until `dispatcher.invokeAction` finishes and handles errors [#59](https://github.com/zowe/zlux-platform/pull/59)
-- Cross-launch via URL has been implemented [#234](https://github.com/zowe/zlux-app-manager/pull/234)
-- Improvements to feature and bookmarking functionality [#30](https://github.com/zowe/tn3270-ng2/pull/30)
-- Added close all + undo settings + hotkeys [#153](https://github.com/zowe/zlux-editor/pull/153)
+- The app server is now able to use more than one certificate authority (CA). This allows the server to validate other server's authenticity by recognizing the CA that another server may have used [#128](https://github.com/zowe/zlux-app-server/pull/128)
+- The ngx-color picker has been replaced by a custom hue selection bar, lightness swatches bar, and color palette, allowing for a more customizable personalization experience [#235](https://github.com/zowe/zlux-app-manager/pull/235)
+- In this version, cross-launch via URL has been implemented, allowing for integration between Zlux and other web applications. This feature enables users to bookmark a set of app2app communication actions (in the form of a URL) that will be executed when opening the webpage [#234](https://github.com/zowe/zlux-app-manager/pull/234)
+- Improvements have been made to the bookmarking functionality in the TN3270 emulator [#30](https://github.com/zowe/tn3270-ng2/pull/30)
+  - Codepages have been reorganized so that the numbers are shown first, making it easier for users to navigate to their favorites
+  - Users can now save connection preferences on a per-user level. Clicking the floppy disk icon saves user settings to that user's scope.
+  - The buttons found in this feature have been realigned  
+- Several features have been added to the Zlux Editor [#153](https://github.com/zowe/zlux-editor/pull/153)
```

Comment:
Sorry, our repo name zlux-editor is misleading. I believe on the Docs site, we refer to it simple as "Editor" or "Zowe Editor" https://docs.zowe.org/stable/user-guide/mvd-using.html#editor